rest_api/rest_api/controller/health.py

```diff
@@ -0,0 +1,126 @@
+from typing import List
+
+import logging
+
+import os
+import pynvml
+import psutil
+
+from pydantic import BaseModel, Field
+
+from fastapi import FastAPI, APIRouter
+
+import haystack
+
+from rest_api.utils import get_app
+from rest_api.config import LOG_LEVEL
+
+logging.getLogger("haystack").setLevel(LOG_LEVEL)
+logger = logging.getLogger("haystack")
+
+
+router = APIRouter()
+app: FastAPI = get_app()
+
+
+class CPUUsage(BaseModel):
+    avg_1m: str = Field(..., description="Average CPU usage over the last 1 minute")
+    avg_5m: str = Field(..., description="Average CPU usage over the last 5 minutes")
+    avg_15m: str = Field(..., description="Average CPU usage over the last 15 minutes")
+
+
+class MemoryUsage(BaseModel):
+    total: str = Field(..., description="Total memory in megabytes")
+    used: str = Field(..., description="Used memory in megabytes")
+    percent: str = Field(..., description="Percentage of used memory")
```

Comment:
Referring to the comment above: here you're specifying the unit in the description, so it should also be ok to simply return the float.